<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWashReagentProbeDlg</class>
 <widget class="BaseDlg" name="AddWashReagentProbeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">AddWashReagentProbeDlg</string>
  </property>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>367</x>
     <y>372</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_btn">
   <property name="geometry">
    <rect>
     <x>497</x>
     <y>372</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>560</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>10</number>
    </property>
    <item row="3" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>受污染物</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="probe_type_cbox">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>R1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QComboBox" name="to_assay_cbox">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="maxVisibleItems">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="from_assay_cbox">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="maxVisibleItems">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>序号</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>污染物</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>试剂针</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QComboBox" name="detergent_type_cbox">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>清洗方式</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="num_edit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>quit_btn</sender>
   <signal>clicked()</signal>
   <receiver>AddWashReagentProbeDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
